src/Makevars.win

```diff
@@ -1 +1,2 @@
+PKG_CXXFLAGS=-std=gnu++0x
```

Comment:
I think this needs to be -std=c++0x in order to be portable to non-GNU compilers.

That said, I believe R has a scheme of defining environment variables in order to do this portably, see: http://cran.r-project.org/doc/manuals/r-release/R-exts.html#Using-C_002b_002b11-code

And that said, I think it's better if we figure out what code requires c++0x and change it (so the package compiles everywhere no matter the compiler). Where in the code were you seeing an error/warning?